views/scss/qti/_qti-base.scss

```diff
@@ -84,20 +84,20 @@ $mainContainer : '.qti-item';
     padding: 15px;
     background: white();
     margin:auto;
-    max-width: 1024px;
+    max-inline-size: 1024px;
 
     &.runtime {
-        min-height: 100%;
+        min-block-size: 100%;
```

Comment:
Note:
 Ideally, all styles not related to ChoiceInteraction should go to this PR: https://github.com/oat-sa/extension-tao-itemqti/pull/2669

But if you think some other changes are needed, you can leave them here, I'll move them later if necessary.